extensions/arcade/sawtooth_battleship/battleship_cli.py

```diff
@@ -227,23 +227,23 @@ def do_init(args, config):
 
     save_config(config)
 
-    wif_filename = config.get('DEFAULT', 'key_file')
-    if wif_filename.endswith(".priv"):
-        addr_filename = wif_filename[0:-len(".priv")] + ".addr"
+    priv_filename = config.get('DEFAULT', 'key_file')
+    if priv_filename.endswith(".priv"):
+        addr_filename = priv_filename[0:-len(".priv")] + ".addr"
```

Comment:
@jjason The ADDR file is being uses as an identifier, and is not a PUB file. I decided to leave as is. The pub file might serve fine as an identifier, but I wasn't sure. Let's leave as is. ;)